services/slam/builtin/cartographer_int_test.go

```diff
@@ -49,7 +49,7 @@ func testCartographerPositionAndMap(t *testing.T, svc slam.Service) {
 	actualMIME, _, pointcloud, err := svc.GetMap(context.Background(), "test", "pointcloud/pcd", nil, false)
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, actualMIME, test.ShouldResemble, "pointcloud/pcd")
-	test.That(t, pointcloud.Size(), test.ShouldEqual, 200)
+	test.That(t, pointcloud.Size(), test.ShouldBeGreaterThanOrEqualTo, 100)
```

Comment:
We can't assume that a point cloud will have a certain number of points. Instead, we estimate that there is a minimum amount of points added.

Reply:
[q] Is it not deterministic, given the same config para and dataset such that we should be able to estimate the exact number of points?